securedrop/create-dev-data.py

```diff
@@ -110,11 +111,12 @@ def create_source_and_submissions(
     # Generate some test submissions
     for _ in range(num_submissions):
         source.interaction_count += 1
+        submission_text = next(submissions)
```

Comment:
looks good! do you want to add special strings in the replies as part of this PR?

Reply:
(Allie did so in 46a5d8783eb1c7495654c4a3f0a2c5519c3bf10f)